compiler/rustc_infer/src/infer/mod.rs

```diff
@@ -535,7 +549,11 @@ pub trait TyCtxtInferExt<'tcx> {
 
 impl<'tcx> TyCtxtInferExt<'tcx> for TyCtxt<'tcx> {
     fn infer_ctxt(self) -> InferCtxtBuilder<'tcx> {
-        InferCtxtBuilder { tcx: self, defining_use_anchor: None, fresh_typeck_results: None }
+        InferCtxtBuilder {
+            tcx: self,
+            defining_use_anchor: DefiningAnchor::Error,
```

Comment:
This used to be the equivalent to `DefiningAnchor::Bubble`.

Reply:
Ah sorry, I totally forgot about this I will look into this asap.